packages/ia-topnav/src/user-menu.js

```diff
@@ -20,6 +20,12 @@ class UserMenu extends DropdownMenu {
     this.username = '';
   }
 
+  get menuClass() {
```

Comment:
Question: Any way to DRY this?

Reply:
Yeah, since they're both subclassed from the same class and they're the only two doing so, I moved it to the super class method instead.